### PR TITLE
Slugs's 404 triggers Sentry reports

### PR DESCRIPTION
**Pull request by Fuzznimp**
## Description

This PR fixes the fact that Sentry got triggered for the `404` errors.

I tested it through the review app.

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [x] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Issues

[CU-h17xb1](https://app.clickup.com/t/h17xb1)

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by Fenntasy** [comment]
Task linked: [CU-h17xb1 [investigate] - Clicking on `homepage` link from slug 404 triggers Sentry reports](https://app.clickup.com/t/h17xb1)

File: src/pages/_error.tsx [hunk 1]
```diff
@@ -37,29 +37,30 @@ const CustomErrorComponent = (
...
 
       return errorInitialProps;
     }
   } else {
-    if (err) {
-      Sentry.captureException(err);
+    if (error) {
+      if (!("statusCode" in error && error.statusCode === 404)) {
+        Sentry.captureException(error);
+      }
```
Comment (Meyclem) [inline src/pages/_error.tsx:63]: This is it.

File: src/pages/api/identities/[id]/index.ts [hunk 2]
```diff
@@ -68,8 +68,10 @@ const get: Handler = async (request, response) => {
       identityId,
     }).catch((error) => {
       span.setDisclosedAttribute("is Identity fetched", false);
-
-      if (error instanceof IdentityNotFoundError) {
+      if (
+        error instanceof IdentityNotFoundError ||
+        error.message === "Invalid UUID format"
+      ) {
```
Comment (Meyclem) [inline src/pages/api/identities/[id]/index.ts:74]: Atm, when the id in the url is a valid UUID but the identity doesn't exist, it throws a 404.
But when it is **not** a valid UUID it throws a 500. I think the fact that use user writes a valid UUID or not should not be treated differently than any other query string.

**Comment by Oluane** [comment]
Well done guys 🎉

**Pull request merged**